script/upgrade_bundler.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
```

Comment:
Maybe I should add some comments here to explain the script.

This line makes the script exit whenever a command is not successful.